### PR TITLE
Add 3 new movie urls

### PR DESCRIPTION
**Pull request by shaunkawano**
Related Issue: https://github.com/konifar/droidkaigi2016/issues/334

## Done

I added 3 new movie urls: https://github.com/konifar/droidkaigi2016/issues/334#issuecomment-197222077 (finally 🙇 )

### CODE REVIEW

File: app/src/main/res/raw/sessions_ar.json [hunk 1]
```diff
@@ -984,7 +984,7 @@
     },
     "language_id": "ja",
     "slide_url": "https://speakerdeck.com/player/45c463463fbe47518576240a53622cdc",
-    "movie_url": "https://www.youtube.com/watch?v=GMAXZPndBo0",
+    "movie_url": "https://www.youtube.com/watch?v=lmti0rNxbmI",
```
Comment (shaunkawano) [inline app/src/main/res/raw/sessions_ar.json]: I'm guessing when I added movie urls for the first time I added the wrong one, so here I modified it to add the right one.

File: app/src/main/res/raw/sessions_en.json [hunk 2]
```diff
@@ -984,7 +984,7 @@
     },
     "language_id": "ja",
     "slide_url": "https://speakerdeck.com/player/45c463463fbe47518576240a53622cdc",
-    "movie_url": "https://www.youtube.com/watch?v=GMAXZPndBo0",
+    "movie_url": "https://www.youtube.com/watch?v=lmti0rNxbmI",
```
Comment (shaunkawano) [inline app/src/main/res/raw/sessions_en.json]: Same as above

**Comment by konifar** [comment]
👀

**Comment by konifar** [comment]
@shaunkawano Thanks!!
And sorry, it's CI setting matter. I'll check and fix this after work.

**Comment by shaunkawano** [comment]
@konifar Thanks for the review!

Yes, I realized that Circle CI failed due to this: http://stackoverflow.com/questions/29063968/plugin-is-too-old-please-update-to-a-more-recent-version-or-set-android-daily; I'm guessing it should be fixed by updating Gradle verson. http://stackoverflow.com/a/29064084/5449688

**Comment by konifar** [comment]
@shaunkawano Oh, thanks. I appreciate that you fix this if you have a time 🙇

**Comment by konifar** [comment]
Could you merge master to this branch?

**Comment by shaunkawano** [comment]
😄 Circle CI check successful!

**Comment by konifar** [comment]
Thanks so much!!

**Pull request merged**